CHANGELOG.md

```diff
@@ -30,6 +30,9 @@ and this project adheres to
 
 #### Deprecated
 #### Removed
+- Disallow different lhist bounds in a single map
+  - [#2204](https://github.com/iovisor/bpftrace/pull/2204)
+
```

Comment:
Move this to "Fixed" please. It has never been supported so it shouldn't be in "Removed".